pkg/controller/controller.go

```diff
@@ -38,6 +38,7 @@ import (
 	"k8s.io/client-go/util/workqueue"
 )
 
+// CSIAttachController contain required interfaces of handlers, eventrecorder, workqueue and listers.
```

Comment:
contain -> contains

And IMO it's not interesting what *exactly* it contains (handlers, ...), but what is it good for, so "CSIAttachController is a controller that attaches / detaches CSI volumes using provided Handler interface." Or something else in this direction.

Reply:
Sure, Done.